src/com/xilinx/rapidwright/rwroute/RWRoute.java

```diff
@@ -289,14 +299,79 @@ private void determineRoutingTargets(){
 				System.err.println("ERROR: Unknown net " + net.toString());
 			}
 		}
-		if(this.config.isVerbose()) this.printConnectionSpanStatistics();
+	}
+	
+	/**
+	 * Deals with the conflicted nets.
+	 * Note: this is customized for the RapidStream use case. Other users will need to adapt this method.
+	 * @param design The design to route.
+	 * @param conflictNets A set of conflicted nets.
+	 */
+	private void handleConflictNets(Design design, Set<Net> conflictNets) {
```

Comment:
I am confused by the arguments of this method.  In the constructor, the `design` to be routed is provided.  There is also a private field for `conflictNets`.  Why does the caller need to provide these again?